setup.py

```diff
@@ -31,10 +31,16 @@
     'requests>=2.13.0',
     'scipy>=0.18.1',
     'six>=1.10.0',
-    'tensorflow>=1.0.0',
     'Werkzeug>=0.11.15',
 ]
 
+# only add tensorflow as a requirement if it is not already provided.
```

Comment:
That was my intention with this comment. Let me know if that isn't clear enough.